mllib-local/benchmarks/BLASBenchmark-jdk11-results.txt

```diff
@@ -2,251 +2,337 @@
+f2j                                                 387            390           3        129.1           7.7       1.0X
+java                                                387            390           1        129.1           7.7       1.0X
+native                                              384            387           1        130.2           7.7       1.0X
 
 
 ================================================================================================
 dgemm[N,N]
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 11.0.11+9-LTS on Linux 5.4.0-1046-azure
-Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
+OpenJDK 64-Bit Server VM 11.0.13+8-LTS on Linux 5.11.0-1022-azure
+Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 dgemm[N,N]:                               Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-f2j                                                 914            956          25       1094.1           0.9       1.0X
-java                                                347            370          11       2881.8           0.3       2.6X
+f2j                                                 744            749           5       1343.4           0.7       1.0X
+java                                                286            293           6       3497.6           0.3       2.6X
+native                                              685            687           2       1459.0           0.7       1.1X
 
 
 ================================================================================================
 dgemm[N,T]
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 11.0.11+9-LTS on Linux 5.4.0-1046-azure
-Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
+OpenJDK 64-Bit Server VM 11.0.13+8-LTS on Linux 5.11.0-1022-azure
+Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
 dgemm[N,T]:                               Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-f2j                                                 927            968          22       1078.4           0.9       1.0X
-java                                                348            370          10       2872.6           0.3       2.7X
+f2j                                                 773            780           6       1294.1           0.8       1.0X
+java                                                288            296           6       3473.2           0.3       2.7X
+native                                              708            711           2       1412.4           0.7       1.1X
```

Comment:
According to SPARK-35150, this means that native libraries are installed somehow, but the performance is worse than Java.
```
If native libraries are not properly configured in the system, the Java implementation (javaBLAS) will be used as fallback option.
```